internal/integration_test/asm/amd64_debug/golang_asm.go

```diff
@@ -508,9 +508,9 @@ var castAsGolangAsmInstruction = [...]obj.As{
 	amd64.ORQ:       x86.AORQ,
 	amd64.POPCNTL:   x86.APOPCNTL,
 	amd64.POPCNTQ:   x86.APOPCNTQ,
-	amd64.PSLLL:     x86.APSLLL,
+	amd64.PSLLD:     x86.APSLLL,
```

Comment:
maybe trailing comment when convention changes like this happen, ex D(ouble) to L(forget why it isn't double ;)) otherwise, it is hard to know if this is a typo or intentional and sometimes typos happen.

Reply:
ok yeah will do from the next time!